week0003/hoangdangduy/java/Week0003.java

```diff
@@ -0,0 +1,157 @@
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+class ModelCustomer {
+	private String action;
+	private String name;
+	private double money;
+
+	public String getAction() {
+		return action;
+	}
+
+	public void setAction(String action) {
+		this.action = action;
+	}
+
+	public String getName() {
+		return name;
+	}
+
+	public void setName(String name) {
+		this.name = name;
+	}
+
+	public double getMoney() {
+		return money;
+	}
+
+	public void setMoney(double money) {
+		this.money = money;
+	}
+
+	@Override
+	public String toString() {
+		return "ModelCustomer [action=" + action + ", name=" + name + ", money=" + money + "]";
+	}
+
+}
+
+public class Week0003 {
+
+	public List<ModelCustomer> readCustomerFromFile() {
+		String folderPath = null;
+		List<ModelCustomer> listAct = new ArrayList<>();
+
+		try {
+			folderPath = new java.io.File(".").getCanonicalPath();
+		} catch (IOException e1) {
+			e1.printStackTrace();
+		}
+
+		Path path = Paths.get(folderPath, "test_case_01.input");
```

Comment:
The requirement was reading from STDIN, not from a file! There was a file redirection, not directly reading file in code. If you hard-code the file name, you won't be able to test with other test cases and this program will not work in production!